src/core/net/netif.cpp

```diff
@@ -399,7 +399,7 @@ Error Netif::SubscribeExternalMulticast(const Address &aAddress)
 
     for (const MulticastAddress *cur = &linkLocalAllRoutersAddress; cur; cur = cur->GetNext())
     {
-        VerifyOrExit(cur->GetAddress() != aAddress, error = kErrorInvalidArgs);
+        VerifyOrExit(cur->GetAddress() != aAddress, error = kErrorRejected);
```

Comment:
Should we also update the new `Rejected` error in `netif.hpp` docs for `SubscribeExternalMulticast()`?
Also in public OT header `ip6.h` for `otIp6SubscribeMulticastAddress()`?

Reply:
doc updated.